### PR TITLE
React 16

### PR DESCRIPTION
**Pull request by R0nd**
Updated react and react-dom to 16.0 as per #1 
Also added eslint packages since the test script is dependent on them. You should probably add a project-level eslint config.

### CODE REVIEW

**Comment by R0nd** [comment]
BTW I noticed that you have react and react-dom installed as dev-dependencies, might want to move them to production dependencies.

File: package.json [hunk 1]
```diff
@@ -20,11 +20,14 @@
   "homepage": "https://github.com/drom/smith-chart#readme",
   "devDependencies": {
     "@vx/responsive": "0.0.120",
+    "babel-eslint": "^8.0.1",
```
Comment (drom) [inline package.json]: I don't use babel

File: package.json [hunk 2]
```diff
@@ -20,11 +20,14 @@
...
     "browserify": "^14.3.0",
     "eslint": "^3.19.0",
+    "eslint-config-google": "^0.9.1",
+    "eslint-plugin-react": "^7.4.0",
```
Comment (drom) [inline package.json]: I don't use JSX

File: package.json [hunk 3]
```diff
@@ -20,11 +20,14 @@
...
     "http-server": "^0.10.0",
     "mocha": "^3.3.0",
-    "react": "^16.0.0-alpha.12",
-    "react-dom": "^16.0.0-alpha.12"
+    "react": "^16.0.0",
+    "react-dom": "^16.0.0"
```
Comment (drom) [inline package.json:28]: This good

**Review by drom — COMMENTED** [review]
This package don't require React. You can use any React / Preact / Inferno.
First User provides React.createElement - like method and it returns component.

**Pull request merged**